app/views/posts/_list.haml

```diff
@@ -1,12 +1,14 @@
+-# locals: ( posts:, table_class: '', hide_continuity: false, show_unread_count: false, check_box_name: false )
```

Comment:
```suggestion
-# locals: ( posts:, table_class: '', hide_continuity: false, show_unread_count: false, check_box_name: nil )
```
(Seems better to combine nil and string than false and string - this field is a string in the list_item it's passed into)